spec/std/isa/isa/fp.idl

```diff
@@ -138,7 +138,7 @@ function check_f_ok {
     raise an exception.
   }
   body {
-    if (MUTABLE_MISA_F && CSR[misa].F == 0) {
+    if (MUTABLE_MISA_F && ((!MISA_CSR_IMPLEMENTED && !implemented?(ExtensionName::F)) || (CSR[misa].F == 0))) {
```

Comment:
This change is unnecessary. If the F bit is mutable, then the CSR is implemented.